repair/repair.hh

```diff
@@ -178,10 +178,10 @@ struct get_sync_boundary_response {
 using get_combined_row_hash_response = repair_hash;
```

Comment:
> The problem is that repair_meta is addressed by IP. In replace-with-same-ip scenario it's possible that the new node bootstrap/repair process is started before the gossiper events reach some of the nodes, and then later when these events eventually come, they remove the new repair_meta, which is used by the ongoing repair process.


Why weren't we hitting this before? Is it because the wait in topology_state_load prevented repair from being started until these events reached everyone?